types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
If the class is on the default export then you can write `import TransitionGroup from "..."`